native/CMakeLists.txt

```diff
@@ -515,6 +517,8 @@ cocos_source_files(
     cocos/platform/interfaces/modules/ISystem.h
     cocos/platform/interfaces/modules/ISystemWindow.h
     cocos/platform/interfaces/modules/IVibrator.h
+    cocos/platform/interfaces/modules/XRCommon.h
+    cocos/platform/interfaces/modules/IXRInterface.h
```

Comment:
Should include them only use xr?

Reply:
This is an interface class. It also participates in compilation when there is no XR, so there is no need to use XR macros everywhere.

Reply:
XR static library has not been submitted in cocos engine external warehouse. This library will be required when "use_xr" is opened. Now it is in non XR state, and these interface classes are required to participate in compilation. If these interface classes are not available at all, then you need to use "use_xr" to include them in all places where XR is used. At this time, if the engine modifies some functions, but the XR macro is not opened, you may forget to adapt XR.